ckanext/xloader/templates/package/resource_read.html

```diff
@@ -0,0 +1,12 @@
+{% ckan_extends %}
+
+{% block action_manage %}
+  {{ super() }}
+  {%
+    if (h.is_xloader_format(res.format) or res.datastore_active)
+    and h.check_access('package_update', {'id':pkg.id })
+    and res.url_type not in h.datastore_rw_resource_url_types()
```

Comment:
This helper does not exist in any currently released version of CKAN. It should only be used if present.

Reply:
last condition now replaced with new helper which calls `datastore_rw_resource_url_types` in try/catch.